dbt_subprojects/dex/models/_log_decoded_trades/arbitrum/decoded-in-flight/uniswap-v2/uniswap_v2_forks_base_trades_arbitrum.sql

```diff
@@ -0,0 +1,61 @@
+{{ config(
+
+        schema = 'dex_mass_decoding_arbitrum',
+        alias = 'uniswap_v2_base_trades',
+        partition_by = ['block_month'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['tx_hash', 'evt_index'],
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
+        )
+}}
+
+WITH all_decoded_trades AS (
+    {{
+        uniswap_v2_forks_trades(
+            blockchain = 'arbitrum'
+            , version = '2'
+            , project = 'null'
+            , Pair_evt_Swap = ref('uniswap_v2_pool_decoding_arbitrum')
+            , Factory_evt_PairCreated = ref('uniswap_v2_factory_decoding_arbitrum')
+        )
+    }}
+)
+
+SELECT  uniswap_v2_base_trades.blockchain
+        , contracts.namespace AS project
+        , uniswap_v2_base_trades.version
```

Comment:
Using `evms.contracts:namespace` and fork version to populate `project` and `version` fields.

Duped produced due to same contract address existed with different namespaces in `evm.contracts`.

Error info:
https://github.com/duneanalytics/spellbook/actions/runs/11200395271/job/31133983915?pr=6630#step:12:54

Duped details:
https://dune.com/queries/4131970